src/handlebars-utils.js

```diff
@@ -72,8 +72,9 @@ HandlebarsUtils.ESCAPE_EXPRESSION = 2; // {{expression}}
 HandlebarsUtils.escapeExpressionRegExp = /^\{\{~?\s*@?\s*([^\}\{~]+)~?\}\}(?!})/;
 
 /* '{{' '~'? '>' '\s'* ('not \s, special-char'+) '\s'* 'not ~{}'* non-greedy '}}' and not follow by '}' */
+/* slash should be allowed */
 HandlebarsUtils.PARTIAL_EXPRESSION = 3; // {{>.*}}
-HandlebarsUtils.partialExpressionRegExp = /^\{\{~?>\s*([^\s!"#%&'\(\)\*\+,\.\/;<=>@\[\\\]\^`\{\|\}\~]+)\s*[^~\}\{]*~?\}\}(?!})/;
+HandlebarsUtils.partialExpressionRegExp = /^\{\{~?>\s*([^\s!"#%&'\(\)\*\+,\.;<=>@\[\\\]\^`\{\|\}\~]+)\s*([^~\}\{]*)~?\}\}(?!})/;
```

Comment:
em... i guess a comment is not needed: i believe handlebars allows using `/` among other chars in partialExpression. i think it's actually a bug when `/` was disallowed.

here's an example in which `express-handlebars` is using `/`, and it functions properly with the vanilla handlebars.
https://github.com/ericf/express-handlebars/blob/master/examples/advanced/views/echo.handlebars

I guess it's more like a TODO, can you pls check if your regexp is over restrictive, or if it is intended?